source/bson-binary-vector/tests/README.md

```diff
@@ -101,19 +101,17 @@ means that two single bit vectors in which 7 bits are ignored do not match unles
 server does.
 
 ```python
-b1 = Binary.from_vector([0b10000000], BinaryVectorDtype.PACKED_BIT, padding=7)
-assert b1 == Binary(b'\x10\x07\x80', subtype=9) # This is effectively a roundtrip.
-v1 = Binary.as_vector(b1)
+b1 = Binary(b'\x10\x07\x80', subtype=9)
+b2 = Binary(b'\x10\x07\xff', subtype=9)
```

Comment:
I like this. Thank you for catching the oversight! I'd still like the example of equality though.

`assert b1 == Binary.from_vector([0b10000000], BinaryVectorDtype.PACKED_BIT, padding=7)`

Reply:
Discussed offline. Used the form `b3 = Binary.from_vector([0b10000000], BinaryVectorDtype.PACKED_BIT, padding=7)` to express the data in the base-literal notation.